src/v/redpanda/admin/api-doc/broker.json

```diff
@@ -145,6 +145,50 @@
                 }
             ]
         },
+        {
+            "path": "/v1/brokers/{id}/cancel_incoming_partition_moves",
```

Comment:
Having two endpoints for cancelling in different directions feels a bit unnatural, maybe make direction a parameter? And I guess cancelling moves related to a node in both directions would be useful too (and making a third endpoint for that is a bit too much :)).

Reply:
I think the third one makes sense from a user perspective. Cancel all operations on X node. For customer <large security firm> when they have a miss behaving node they just want to stop all movement to that node. The third can just make 2 proxy calls (incoming first and outgoing after)

Reply:
@mattschumpert any thoughts on this one? Or anyone else we can ping with more experience around REST API UX?

Reply:
Agree with @emaxerrno completely. Let's not make the customer do to API calls to quiesce the emergency.  Cancel incoming could be useful in a disk low situation though (where the autobalancer can't keep up).

Reply:
just to make sure i understand we will have 3 APIs:

```
POST /v1/brokers/{id}/cancel_incoming_partition_moves
POST /v1/brokers/{id}/cancel_outgoing_partition_moves
POST /v1/brokers/{id}/cancel_all_partition_moves
```

?